xml/Microsoft.Aspnet.Snapin/AspNetManagementUtility.xml

```diff
@@ -223,13 +279,37 @@
         <Parameter Name="type" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="hWnd">To be added.</param>
-        <param name="text">To be added.</param>
-        <param name="caption">To be added.</param>
-        <param name="type">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="hWnd">A handle to the owner of the message box to create. If hWnd is null (Nothing in Visual Basic), the message box has no owner window.</param>
+        <param name="text">The message for the message box to display.</param>
+        <param name="caption">The message box title.</param>
+        <param name="type">The contents and behavior of the message box.</param>
+        <summary>Creates, displays, and operates a message box in the ASP.NET snap-in for the Microsoft Management Console (MMC).</summary>
+        <returns>IDCANCEL, if the message box has a **Cancel** button and either ESC is pressed or **Cancel** is clicked. If the message has no **Cancel** button, pressing ESC has no effect.  
+If the function fails, 0.  
+If the function succeeds, the return value is one of the following menu-item values:  
+
+
+
+| Return value | Description |
+| --- | --- |
+| IDABORT | **Abort** was clicked. |
+| IDCANCEL | **Cancel** was clicked. |
+| IDCONTINUE | **Continue** was clicked. |
+| IDIGNORE | **Ignore** was clicked. |
+| IDNO | **No** was clicked. |
+| IDOK | **OK** was clicked. |
+| IDRETRY | **Retry** was clicked. |
+| IDTRYAGAIN | **Try Again** was clicked. |
+| IDYES | **Yes** was clicked. |</returns>
```

Comment:
Will this markdown render inside a ‘returns’ section? I thought it wouldn’t?

Reply:
It seems to be working on preview and since this is dead content, we don't need to worry about IntelliSense:
![image](https://user-images.githubusercontent.com/12971179/48867000-94f22600-ed89-11e8-8bd4-6e16366b49b7.png)

Reply:
Sorry - looks like I’m wrong! I’ve PR’d corrections to convert returns sections links from markdown to href’s before...perhaps those were unnecessary! 😅